server/app/views/admin/apikeys/ApiKeyCredentialsView.java

```diff
@@ -21,6 +21,12 @@
 public final class ApiKeyCredentialsView extends BaseHtmlView {
   private final AdminLayout layout;
 
+  private static final String CREDENTIALS_DESCRIPTION =
+      "Please copy your API key and it store it somewhere secure. This is your only"
+          + " opportunity to copy the secret from CiviForm, if you "
+          + "refresh the page or navigate away you will not be able to recover the"
```

Comment:
Move the trailing space before this to a leading space here.

Reply:
done